nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -56,6 +57,16 @@ class AbstractRNNTDecoding(ABC):
                 Each value in the list (Ti) is a torch.Tensor (U), representing 1 or more targets from a vocabulary.
                 U is the number of target tokens for the current timestep Ti.
 
+            compute_timestamps: A bool flag, which determines whether to compute the character/subword, or
+                word based timestamp mapping the output log-probabilities to discrete intervals of timestamps.
+                The timestamps will be available in the returned Hypothesis.timestep as a dictionary.
+
+            rnnt_timestamp_type: A str value, which represents the types of timestamps that should be calculated.
+                Can take the following values - "char" for character/subword time stamps, "word" for word level
+                time stamps and "all" (default), for both character level and word level time stamps.
```

Comment:
Does it take significant time to always compute both the word and char level time stamps?

Reply:
Nope, that's why default is all mode, but problem is it only works for subword at the moment so i offer option to skip for WPE based Tokenizer